blocks/lead-art-block/features/leadart/default.jsx

```diff
@@ -142,6 +142,11 @@ export const LeadArtPresentation = (props) => {
 							resizedOptions={{ auth: lead_art.auth[RESIZER_APP_VERSION], smart: true }}
 							resizerURL={RESIZER_URL}
 						/>
+						{isOpen ? (
+							<Button type="button" onClick={toggleFullScreen}>
```

Comment:
This should have an `accessibilityLabel` prop with the value `phrases.t("lead-art-block.fullscreen-exit")`.

Reply:
The button above on line `124` needs the prop to be fixed as well, from `label` to `accessibilityLabel`.

Reply:
Did the changes